src/app/shared/model/UserTerms.model.ts

```diff
@@ -1,5 +1,5 @@
 export interface UserTermsModel {
```

Comment:
change user terms model data types to any type for handle any type of data in API requests